### PR TITLE
Validator and account management updates

### PR DESCRIPTION
**Pull request by CloudEllie**
- [x] Update Validators page to reflect single validator processes
- [x] Clarify that Pro League wardens' submissions are sent directly to findings repo
- [x] Move user accounts page out of wardens section 
- [x] add GH verification steps to account management page

### CODE REVIEW

**Review by itsmetechjay — APPROVED** [review]
These changes look great to me!

File: roles/certified-contributors/sr-backstage-wardens.md [hunk 1]
```diff
@@ -16,6 +16,8 @@ The minimum criteria to become a Certified SR are as follows:
 
 Once you meet the eligibility criteria, submit a [Help Desk Request](https://code4rena.com/help/) to request the SR role, and C4 staff will get you set up.
 
+You should also verify your Github account via your Code4rena account management screen, in order to access private repos.
```
Comment (sockdrawermoney) [inline roles/certified-contributors/sr-backstage-wardens.md:19]: Maybe a direct link to the settings page here would be nice.

File: structure/account-management.md [hunk 2]
```diff
@@ -42,6 +44,25 @@ If you can't see most channels in the Code4rena Discord server, your Discord ver
...
+### Github verification (optional)
+
+Certified and SR wardens should verify their Github accounts in order to access private repos. 
+
+**Note:** Ensure that you are logged in to the correct GitHub account before starting the verification process.
+
+To verify your GitHub account on Code4rena, follow these steps:
+
+1. Log in to your Code4rena account.
+2. Go to the **Account Settings** page.
```
Comment (sockdrawermoney) [inline structure/account-management.md:56]: Same here, suggest a link since we can provide that

**Review by sockdrawermoney — APPROVED** [review]

File: roles/certified-contributors/sr-backstage-wardens.md (hunk 1 above)
Comment (CloudEllie) [reply roles/certified-contributors/sr-backstage-wardens.md:19]: good catch - added

**Pull request merged**